wp/lib/bap_wp/src/environment.ml

```diff
@@ -455,7 +464,7 @@ let get_smtlib_compat (env : t) : bool = env.smtlib_compat
 let init_stack_ptr (env : t) : Constr.z3_expr -> Constr.z3_expr =
   let ctx = get_context env in
   let arch = get_arch env in
-  let sort = arch |> Arch.addr_size |> Size.in_bits |> BV.mk_sort ctx in
+  let sort = arch |> Theory.Target.data_addr_size |> BV.mk_sort ctx in
```

Comment:
Would it be a good idea to add a comment here so we know this is in bits?

Reply:
Maybe? This feels like the responsibility of the BAP API, not us.

Ivan claims that bits is better all arounds, since apparently there are systems for which a byte is not 4 bits (crazy).

https://stackoverflow.com/a/18743555/2059388

Reply:
Sounds good. We can keep this if you feel it's better.

Reply:
I don't think this warrants a comment, since it's making use of the API as intended.